lib/ssh_scan/api.rb

```diff
@@ -184,6 +208,10 @@ def self.run!(options = {}, &block)
         set :logger, Logger.new(STDOUT)
         set :job_queue, JobQueue.new()
         set :db, SSHScan::Database.from_hash(options)
+        set :count_req, 0
+        set :current_avg, 0.0
+        set :requests_per_min, 0
+        set :curr_time, Time.now
```

Comment:
The name `:curr_time` seems confusing here, change to `:window_start_time` or something like that?